chia/wallet/wallet_transaction_store.py

```diff
@@ -65,22 +65,20 @@ async def create(cls, db_wrapper: DBWrapper2):
             await conn.execute(
                 "CREATE INDEX IF NOT EXISTS tx_confirmed_index on transaction_record(confirmed_at_height)"
             )
-
             await conn.execute("CREATE INDEX IF NOT EXISTS tx_created_index on transaction_record(created_at_time)")
```

Comment:
Did you dig into history to understand why these two indices ended up being the same? Has it been like this since the first commit? An alternative explanation would be that this index really was meant to index a different column

Reply:
Yes, I traced it back to 331d774.